package.json

```diff
@@ -9,11 +9,11 @@
   },
   "license": "MIT",
   "engines": {
-    "node": ">= 5.4.1",
+    "node": "10",
     "yarn": ">= 1.17.3"
   },
   "scripts": {
-    "postinstall": "node --preserve-symlinks --preserve-symlinks-main tools/bazel/postinstall-patches.js",
+    "postinstall": "node --preserve-symlinks --preserve-symlinks-main tools/bazel/postinstall-patches.js | ivy-ngcc --properties main",
```

Comment:
It might be good to only do this when running within Bazel. ngcc is not very fast and otherwise we are running ngcc twice. We can tackle this in a follow-up though.